primer/src/Primer/Primitives.hs

```diff
@@ -270,6 +275,10 @@ primFunDef def args = case def of
     [exprToNat -> Just n] ->
       Right $ int $ fromIntegral n
     _ -> err
+  PrimConst -> case args of
+    [x, _] ->
+      Right $ generateIDs x
+    _ -> err
```

Comment:
(Note from after this was merged)

It turns out that this definition is slightly incorrect, for bidirectional reasons. See #1159